experimental/framework/fluid-static/src/fluidContainer.ts

```diff
@@ -0,0 +1,73 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { strict as assert } from "assert";
+import { EventEmitter } from "events";
+import { IAudience } from "@fluidframework/container-definitions";
+import { Container } from "@fluidframework/container-loader";
+import { IFluidLoadable } from "@fluidframework/core-interfaces";
+import {
+    LoadableObjectClass,
+} from "./types";
+import { RootDataObject } from "./rootDataObject";
+
+export class FluidContainer
+    extends EventEmitter
+    implements
+        Pick<Container, "audience" | "clientId" | "close" | "closed" | "connected" >,
+        Pick<RootDataObject, "initialObjects"> {
```

Comment:
>Pick<RootDataObject, "initialObjects">  [](http://example.com/codeflow?start=8&length=39)

Let's not do this. RootDataObject's goal is to provide implementation not to define the API.